plugin/trino-hive-hadoop2/src/test/java/io/trino/plugin/hive/TestHive.java

```diff
@@ -92,6 +92,20 @@ public void testHideDeltaLakeTables()
         throw new SkipException("not supported");
     }
 
+    @Override
+    public void testHideIcebergTables()
+    {
+        assertThatThrownBy(super::testHideIcebergTables)
+                .hasMessageMatching("(?s)\n" +
+                        "Expecting\n" +
+                        " <\\[.*\\b(\\w+.tmp_trino_test_trino_iceberg_table_\\w+)\\b.*]>\n" +
+                        "not to contain\n" +
+                        " <\\[\\1]>\n" +
+                        "but found.*");
+
+        throw new SkipException("not supported");
```

Comment:
I found the same model also in the `testHideDeltaLakeTables()` and I am wondering whether it makes sense to do the skipping at the end of the method.
Can anyone explain the reasoning behind this ?

Reply:
the assertion is so we do not forget to "uncomment" the test when we fix the tested code.
But actually why is this test supposed to fail?

Reply:
seems `testDisallowQueryingOfIcebergTables` is expected to pass, so why the override?